zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/tasks/incident/IncidentUpdateTaskTest.java

```diff
@@ -162,30 +164,53 @@ public CompletionStage<List<ActiveIncident>> getActiveIncidentsByTreePaths(
   /**
    * All tests in this class have the following set up:
    *
-   * <p>A single incident (3), a single process instance (1), and a single flow node (2). The
-   * incident was pending, and now has a single update to set it active.
+   * <p>We deployed a process consisting of a call activity, which calls a process with a single
+   * task.
+   *
+   * <p>An instance of that parent process instance (1) has a call activity instance flow node (2)
+   * with a child process instance (3), with a task flow node (4).
+   *
+   * <p>There is a single incident update by default (5) which is being created.
    *
    * <p>The pending update has a position of 10.
    */
   @Nested
-  final class SingleIncidentTest {
+  final class IncidentTreePathTest {
```

Comment:
:thought_balloon: I realized that it's just better coverage to test with a tree path, as it covers the same cases as the single task, and it's not really more expensive.

Reply:
Unit tests done now :+1: